### DISCUSSION TITLE
Nuxt 3 recommended way to get rid of typescript

### DISCUSSION THREAD

**Discussion by leonixyz** [nuxt/nuxt#18773] [Questions]
`nuxi init <project-name>` creates by default a typescript project.

Which is the recommended way to get rid of typescript completely?

Do you plan to give back users the option to choose whether to use typescript or not, or is everyone just forced to deal with it from nuxt 3 on?

thank you in advance

**Comment by danielroe** [comment]
No TS code is generated in the init stage, nor is TypeScript installed. You can rename your `nuxt.config.ts` to `nuxt.config.js`, if you care about the file extension, and you should be set.
